### DISCUSSION TITLE
Use the zxcvbn algorithm to compute entropy

### DISCUSSION THREAD

**Discussion by 97ryan** [Kunzisoft/KeePassDX#1392] [Ideas]
Entropy displayed in the password manager seems questionable, with both passphrases and randomized passwords.

For example, an EFF long-list word as used by KeePassDX has approximately 13 bits of entropy given that algorithm, but in DX it has ~ 11.92. For passwords, the entropy counter seems even more 'off' given the established zxcvbn algorithm used in a lot of places, like KeePassXC.

I understand entropy is not concrete and it is variable, but it would be cool to have this algorithm implemented for uniformity, as I use both XC and DX.

**Comment by J-Jamet** [comment]
Linked to https://github.com/Kunzisoft/KeePassDX/issues/1373
Currently, I'm not going to recreate a library when the one using is working well. You can check the differences between the KeePassXC code and the zxcvbn library to harmonize the algorithms if you wish, I would be happy to update zxcvbn afterwards in KeePassDX.
